src/components/infinite-scroll/__tests__/InfiniteScroll.test.tsx

```diff
@@ -1,19 +1,21 @@
+import { mount } from 'enzyme';
 import * as React from 'react';
 import sinon from 'sinon';
 
-import InfiniteScroll from '../InfiniteScroll';
+import InfiniteScroll, { Props } from '../InfiniteScroll';
 
 const sandbox = sinon.sandbox.create();
 
 const mockOnLoadMore = jest.fn();
 
 const threshold = 100;
-const propsList = {
+const propsList: Props = {
     isLoading: false,
     hasMore: false,
     useWindow: true,
     onLoadMore: mockOnLoadMore,
     threshold,
+    children: null,
```

Comment:
what do you mean? children were a required prop and those tests were wrong - they did not provide the children (because they were not typed) 
`null` is a valid value for children and these props are only used in tests that don't require children so I think it's ok

Reply:
oh you mean, sort alphabetically? why isn't their a prettier rule in place? feels like it should be